### PR TITLE
Clarify CSS rules.

### PR DESCRIPTION
**Pull request by cramforce**

### CODE REVIEW

**Comment by cramforce** [comment]
@amphtml-savage retry

**Comment by cramforce** [comment]
@amphtml-savage retry

File: spec/amp-html-format.md [hunk 1]
```diff
@@ -187,13 +173,18 @@ These CSS properties are permanently banned:
...
-- `animation`
 - `filter`
 
-`overflow` (and `overflow-y`, etc.) may not be styled as “auto” or “scroll”. No user defined element in an AMP document may have a scrollbar.
+AMP only allows transitions and animations of properties that can be GPU accelerated in common browsers. We currently whitelist: `opacity`, `transform` (also `-vendorPrefix-transform`).
+
+In the following examples `<property>` needs to be in the whitelist above.
+
+- `transition <property>` (Also -vendorPrefix-transition)
+- `@keyframes name { from: {<property>: value} to {<property: value>} }` (also `@-vendorPrefix-keyframes`)
```
Comment (dvoytenko) [inline spec/amp-html-format.md]: Please file a separate issue to whitelist `@keyframes` and `@-vendorPrefix-keyframes`

**Comment by dvoytenko** [comment]
LGTM

**Pull request merged**